src/content/terraform/install-landing.json

```diff
@@ -1,5 +1,21 @@
 {
 	"featuredLearnContent": [],
+	"sidebarMenuItems": [
```

Comment:
Added some content to avoid this looking empty for visual QA.

Reply:
You'll probably want to note in the content task that this is done for them: https://app.asana.com/0/1202520168081556/1202557239529907/f

Reply:
Ah great callout 💖 Personally I'm considering this stubbed in content. Even if the off chance no one wants to change it, I think actual content folks should have another look at it before we consider it "done".